[feature][doc][config]Add missing transaction config in standalone.conf and broker.conf (#15614)
apache/pulsar: site2/docs/reference-configuration.md

|replicatorPrefix| Replicator prefix used for replicator producer name and cursor name pulsar.repl                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           ||

> |transactionBufferClientOperationTimeoutInMills| The transaction buffer client's operation timeout in milliseconds.|3000|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |3000|

Reviewer: two 3000

> |transactionCoordinatorEnabled| Whether to enable transaction coordinator in broker.|false|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |true|

Reviewer: wht false and true

|transactionCoordinatorEnabled| Whether to enable transaction coordinator in broker.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |false|

> |transactionMetadataStoreProviderClassName||org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider|

Reviewer 2: The description is missing

|transactionBufferSnapshotMaxTransactionCount|Transaction Buffer will take a snapshot after the number of transaction operations reaches this value.|1000|

> |transactionBufferSnapshotMinTimeInMillis|Transaction buffer take snapshot interval time (Unit : millisecond)|5000|

Reviewer 2: ```suggestion
|transactionBufferSnapshotMinTimeInMillis|The interval time of transaction buffer takes a snapshot (in milliseconds)|5000|
```

> |transactionBufferSnapshotMinTimeInMillis|The interval time of transaction buffer takes a snapshot (in milliseconds).|5000|

Reviewer 3: ```suggestion
|transactionBufferSnapshotMinTimeInMillis|The interval between two snapshots that the transaction buffer takes (in milliseconds).|5000|
```

> |transactionPendingAckLogIndexMinLag|MLPendingAckStore maintains a ConcurrentSkipListMap pendingAckLogIndex. It stores the position in pendingAckStore as its value and saves a position used to determine whether the previous data can be cleaned up as a key. transactionPendingAckLogIndexMinLag is used to configure the minimum lag between indexes|500|

Reviewer 2: who `stores the position in pendingAckStore as its value and saves a position ....`? can you specify it explicitly?

> |transactionPendingAckLogIndexMinLag|MLPendingAckStore maintains a ConcurrentSkipListMap pendingAckLogIndex. PendingAckLogIndex stores the position in pendingAckStore as its value and saves a position used to determine whether the previous data can be cleaned up as a key. transactionPendingAckLogIndexMinLag is used to configure the minimum lag between indexes (keys).|500|

Reviewer 3: ```suggestion
|transactionPendingAckLogIndexMinLag| Used to configure the minimum lag between `PendingAckLogIndex` (keys). `PendingAckLogIndex` stores the position in `MLPendingAckStore` as its value and saves a position used to determine whether the previous data can be cleaned up as a key.|500|
```

What's the measurement for the lag? milliseconds?

|replicationConnectionsPerBroker|   |16|
|replicationProducerQueueSize|    |1000|
| replicationPolicyCheckDurationSeconds | Duration to check replication policy to avoid replicator inconsistency due to missing ZooKeeper watch. When the value is set to 0, disable checking replication policy. | 600 |

> |transactionCoordinatorEnabled| Whether to enable transaction coordinator in broker.|false|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |true|

Reviewer: true and false
